requests_toolbelt/threaded/thread.py

```diff
@@ -36,6 +36,8 @@ def _handle_request(self, kwargs):
             self._jobs.task_done()
 
     def _make_request(self):
+        if self._jobs is None:
+            return
```

Comment:
I think `_jobs` is expected to be a `queue.Queue`, so this would allow invalid use, potentially hiding bugs in real production code. I've taken a different approach in #253 that always passes a `Queue` objects in tests.